spec/requests/api/go_cardless/go_cardless_spec.rb

```diff
@@ -6,6 +6,13 @@
   let(:usd_amount) { 9.99 }
   let(:gbp_amount) { 11.55 }
 
+  let(:bad_request_errors) do
+    [{
+      code: 422,
```

Comment:
Is this the actual error? If so, I'd be a bit more verbose as to what we've been notified of, and add the tech team sentence to the end of the error.